front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -28,26 +29,41 @@ type StdcmConfigProps = {
   setCurrentSimulationInputs: React.Dispatch<
     React.SetStateAction<StdcmSimulationResult['input'] | undefined>
   >;
+  setStdcmSimulationResults: (stdcmSimulationResults: StdcmV2SuccessResponse | undefined) => void;
+  setSimulationCreationDate: (simulationCreationDate: Date) => void;
 };
 
 const StdcmConfig = ({
   currentSimulationInputs,
   pathProperties,
   setPathProperties,
   setCurrentSimulationInputs,
+  setStdcmSimulationResults,
+  setSimulationCreationDate,
 }: StdcmConfigProps) => {
   const { t } = useTranslation('stdcm');
   const loaderRef = useRef<HTMLDivElement>(null);
 
-  const { launchStdcmRequest, cancelStdcmRequest, currentStdcmRequestStatus } = useStdcm();
+  const { launchStdcmRequest, cancelStdcmRequest, currentStdcmRequestStatus, stdcmV2Results } =
+    useStdcm();
```

Comment:
I think useStdcm should be called in the component StdcmViewV2 directly

Reply:
fix